pkg/storage/my_requests.go

```diff
@@ -0,0 +1,41 @@
+package storage
```

Comment:
I would rename this `requests.go`. It won't always be just "my" requests and other storage files are named after their resource.